src/libs/actions/Policy.js

```diff
@@ -375,9 +375,25 @@ function addMembersToWorkspace(invitedEmailsToAccountIDs, welcomeNote, policyID,
             onyxMethod: Onyx.METHOD.MERGE,
             key: membersListKey,
 
-            // Convert to object with each key clearing pendingAction. We don’t
-            // need to remove the members since that will be handled by onClose of OfflineWithFeedback.
-            value: _.object(accountIDs, Array(accountIDs.length).fill({pendingAction: null, errors: null})),
+            // Convert to object with each key clearing pendingAction, when it is an existing account.
+            // Remove the object, when it is a newly created account.
+            value: _.reduce(
+                accountIDs,
+                (memo, accountID) => {
```

Comment:
not trying to be pedantic, but if name is `memo`, we'll have to read the implementation to understand `reduce`.

What do you think?

```suggestion
                (accountIDsWithClearedPendingAction, accountID) => {
```

Reply:
Yeah, why not. I saw that there are many places where they call accumulator like memo, I decided to be consistent. Let's rename!

Reply:
Done - https://github.com/Expensify/App/pull/22410/commits/3b7448e2a607ddf36be4780d55ea932d6f733a2e

Reply:
yes!